### ISSUE TITLE
Improved search on al folders

### ISSUE THREAD

**Issue by sergioeix** [nextcloud/mail#8966]
### Is your feature request related to a problem? Please describe.

By now if you try to search anything the search is done on the active folder, and if you have like 60 folder ... its anoying to make the same search 60 times.
I know you can search with the nc global search but with nc global search you cant do it by from or to address or with date range.

### Describe the solution you'd like

a check box to search on all folders will be great

**Comment by ChristophWurst** [comment]
Indeed. A global search but within the context of Mail is currently missing. Unified search is not a replacement for this.
